lesson_16/test_homework_16.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+
+from homework_16 import TeamLead, get_figure_area_and_perimeter
+
+
+@pytest.fixture(scope="class")
+def obj():
```

Comment:
```suggestion
def team_lead_example():
```
тут з назвою я б подумав, щось типу таке